src/backend/distributed/citus.sql

```diff
@@ -103,6 +103,27 @@ SET search_path = 'pg_catalog';
 
 /* master_* functions */
 
+CREATE FUNCTION citus_table_size(relation_name text)
```

Comment:
When we want to use table names as an argument, generally regclass type is better option. PostgreSQL automatically resolves regclass to actual relation ids.